.github/workflows/submit.yml

```diff
@@ -0,0 +1,46 @@
+name: Kodi Addon-Submitter
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  kodi-addon-submitter:
+    runs-on: ubuntu-latest
+    name: Kodi addon submitter
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Generate distribution zip and submit to official kodi repository
+        id: kodi-addon-submitter
+        uses: xbmc/action-kodi-addon-submitter@v1.1
+        with:
+          kodi-repository: repo-scripts
+          kodi-version: matrix
+          addon-id: script.service.checkpreviousepisode
+          kodi-matrix: true # Submit a Python 2/3 compatible addon to matrix branch in addition to the target branch
```

Comment:
Isn't this the setup for submitting BOTH a Leia and Matrix addon?  And aren't we just submitting a Matrix one here?

Reply:
So far it worked for me, only submitting to matrix. As `kodi-version` is set to `matrix`

Reply:
`kodi-matrix` can be removed if the target branch is matrix. This is only used if you're submitting to some other branch and submitting to matrix as well (it automatically adjusts addon.xml).

Reply:
Btw, I've been working into creating a set of recipes for the majority of use cases.

a) Your addon is fully py2 and py3 compliant and only needs addon.xml to be re-written to submit to both branches: https://github.com/enen92/script.tubecast/tree/master/.github/workflows

b) Your addon has different versions for each kodi version (eg. py3 only for matrix, py2 only for leia). Configuration using named branches: https://github.com/XBMC-Addons/plugin.audio.radio_de

There are a few issues to iron out (mostly figuring out how to avoid some issues using the github action syntax) and some documentation (step by step) to prepare but it seems to be working well

Reply:
Ok in this case I can't see it being likely the Python 2 / Leia version will every really need any further work (it's just a utility with no external scraping issues or whatever) - so this is really just a Python 3 continuation for Matrix and beyond.  

Thus I will merge this, get rid of the unnecessary `kodi-matrix` - prep up my secrets and tag a release.

Looking forward to using this, the whole git schmozzle with sub branches etc was always a mental barrier to the whole process of submitting.  Yaml actions, especially ones that are at least pretty much the same across addons, seem a much better way forward.